lib/puppet/face/certregen.rb

```diff
@@ -47,6 +48,15 @@
     end
   end
 
+  action(:crl) do
+    summary 'Update the lastUpdate and nextUpdate field for the CA CRL'
```

Comment:
Could this be a more user friendly description? As an end user, I may not know what those terms mean or what the effect of this change would be.